rust_snuba/src/processors/utils.rs

```diff
@@ -31,3 +32,10 @@ where
     };
     Ok(seconds_since_epoch.timestamp() as u32)
 }
+
+#[derive(Debug, Deserialize, JsonSchema, Default, Serialize)]
+pub struct StringToIntDatetime(
+    #[serde(deserialize_with = "ensure_valid_datetime")]
+    #[schemars(with = "String")]
```

Comment:
Is this how you initially generated the schema? How come we're checking this change in permanently?

Reply:
because there's a test that asserts the two schemas are the same (or at least compatible), so we have to be able to generate schemas